src/main/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionService.java

```diff
@@ -314,26 +348,38 @@ Map<String, String> getPostComputePalaemonTokens(PalaemonSessionRequest request)
         // encryption
         Map<String, String> encryptionTokens = getPostComputeEncryptionTokens(request);
         if (encryptionTokens.isEmpty()) {
-            throw new Exception("Failed to get post-compute encryption tokens - taskId: " + taskId);
+            // This branch should never happen
```

Comment:
Why :) ?

Reply:
According to `getPostComputeEncryptionTokens`, `encryptionTokens` should always have at least `RESULT_ENCRYPTION` and `RESULT_ENCRYPTION_PUBLIC_KEY` keys. It's more like a safeguard, if `getPostComputeEncryptionTokens` is ever modified.